test/cli/detect_breakends_cli_test.cpp

```diff
@@ -0,0 +1,87 @@
+#include <seqan3/alphabet/nucleotide/dna5.hpp>
+#include <seqan3/io/sequence_file/input.hpp>
+
+#include "cli_test.hpp"
+
+TEST_F(cli_test, no_options)
+{
+    cli_test_result result = execute_app("detect_breakends");
+    std::string expected
+    {
+            "detectJunctions - Detect junctions in a read alignment file\n"
+            "===========================================================\n"
+            "    Try -h or --help for more information.\n"
+    };
+    EXPECT_EQ(result.exit_code, 0);
+    EXPECT_EQ(result.out, expected);
+    EXPECT_EQ(result.err, std::string{});
+}
+
+TEST_F(cli_test, fail_no_argument)
+{
+    cli_test_result result = execute_app("detect_breakends", "-v");
+    std::string expected
+    {
+        "[Error] Unknown option -v. In case this is meant to be a non-option/argument/parameter, please specify "
+        "the start of non-options with '--'. See -h/--help for program information.\n"
+    };
+    EXPECT_NE(result.exit_code, 0);
+    EXPECT_EQ(result.out, std::string{});
+    EXPECT_EQ(result.err, expected);
+}
+
+TEST_F(cli_test, with_arguments)
+{
+    cli_test_result result = execute_app("detect_breakends",
+                                         data("converted_bam_shorted.sam"),
+                                         "detect_breakends_out_short_10.fasta");
```

Comment:
Is this an output file? If yes then its content should also be checked.

Reply:
Oh, yes you're right. 
I just see that there is an output in the std::out and one in a .fasta file.
Could you rename it into:
```suggestion
                                         "detect_breakends_insertion_file_out.fasta");
```
But I think with our small example the file is empty. I will try to figure out a better example, that we can check the content aswell.
Perhaps you are now testing whether the file is empty so that the content check is already there and we just have to adjust it later.